OCA/hr-attendance
# [16.0][ADD] hr_attendance_ip_check: IP-based attendance validation

[16.0][ADD] hr_attendance_ip_check: IP-based attendance validation

This module extends HR attendance to validate check-in/check-out based on IP address.

Features:
- IP whitelist configuration
- Restrict attendance actions to allowed IPs
- Easy enable/disable through settings

Test scenario:
1. Install module
2. Configure allowed IPs in Settings
3. Try check-in from allowed and blocked IPs
